samples/graph-appcatalog-lifecycle/csharp/README.md

```diff
@@ -0,0 +1,64 @@
+﻿### AppCatalog sample
+
+Bot Framework v4 AppCatalog sample.
+
+This sample shows a AppCatalog bot and demonstrates teamsApp lifecycle in catalog followed by commands given to Bot.
+
+## Prerequisites
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+  B) Or from Visual Studio
+
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `graph-appcatalog-lifecycle/csharp` folder
+  - Select `AppCatalogSample.sln` file
+  - Press `F5` to run the project
+  
```

Comment:
Let's include reference to setting up auth part of Bot. Link to documentation should be helpful.